proto/sdk/alpha/alpha.proto

```diff
@@ -122,6 +127,24 @@ service SDK {
             get: "/alpha/player/connected"
         };
     }
+
+    // Gets a Counter. Returns NOT_FOUND if the counter does not exist.
+    rpc GetCounter(GetCounterRequest) returns (Counter) {
+      option (google.api.http) = {
+        get: "/v1alpha1/{name=counters/*}"
+      };
+      option (google.api.method_signature) = "name";
+    }
+
+    // Updates a Counter. Returns INVALID_ARGUMENT if the name of the counter
+    // is non-empty and does not equal the existing name.
+    rpc UpdateCounter(UpdateCounterRequest) returns (Counter) {
```

Comment:
Look at the design doc, it defines the SDK operations as `CountIncrement`, `CountDecrement`, etc. Just want to confirm, you are grouping them together as the `UpdateCounter` to follow the AIP standard, right?

Reply:
Correct yes